src/controllers/__tests__/sessionLockController.test.ts

```diff
@@ -150,7 +207,80 @@ describe("SessionLockController#lockUserSession", () => {
 
     const controller = new SessionLockController(
       mockRedisSessionStorage,
-      mockRedisUserMetadataStorage
+      mockRedisUserMetadataStorage,
+      mockLollipopService
+    );
+
+    const response = await controller.lockUserSession(req);
+    response.apply(res);
+
+    expect(res.status).toHaveBeenCalledWith(500);
+  });
+
+  it("should fail when get assertionRef return left", async () => {
+    const req = mockReq({ params: { fiscal_code: aFiscalCode } });
+    const res = mockRes();
+
+    mockGetLollipop.mockImplementationOnce(async () => E.left("any error"));
+
+    const controller = new SessionLockController(
+      mockRedisSessionStorage,
+      mockRedisUserMetadataStorage,
+      mockLollipopService
+    );
+
+    const response = await controller.lockUserSession(req);
+    response.apply(res);
+
+    expect(res.status).toHaveBeenCalledWith(500);
```

Comment:
We could add a check like 
```ts
expect(mockRevokePreviousAssertionRef).not.toHaveBeenCalled();
```

Reply:
done https://github.com/pagopa/io-backend/pull/995/commits/1216de4e971ce08cf7c7b7b5f9939dfdc4845153